### ISSUE TITLE
The output are always an black picture

### ISSUE THREAD

**Issue by shichaosuper** [shekkizh/FCN.tensorflow#24]
Hello!
I just run your code for 10 min, and the loss quickly converged to 3. But when I goes to the tensorboard,  the prediction are always an black picture. Why this happen? Is there anything wrong with the parameter?
Thanks!

**Comment by 906476903** [comment]
this is the ground truth
![Uploading 1.jpg…]()

**Comment by shichaosuper** [comment]
this is the ground truth
![1](https://cloud.githubusercontent.com/assets/13098609/24959946/3656cd46-1fcf-11e7-8be0-2ba9f69dddbe.jpg)
this is the prediction
![2](https://cloud.githubusercontent.com/assets/13098609/24959967/482848ce-1fcf-11e7-8c43-b59979ea970a.png)

and this is the prediction


Step: 1730, Train_loss:3.34437
Step: 1740, Train_loss:3.3644
Step: 1750, Train_loss:2.99994
Step: 1760, Train_loss:3.33076
Step: 1770, Train_loss:2.73643
Step: 1780, Train_loss:3.85039
Step: 1790, Train_loss:3.56678
Step: 1800, Train_loss:3.60958
Step: 1810, Train_loss:3.84591
Step: 1820, Train_loss:2.71663
Step: 1830, Train_loss:5.04416
Step: 1840, Train_loss:2.36329
Step: 1850, Train_loss:2.62359
Step: 1860, Train_loss:4.41742
Step: 1870, Train_loss:2.94478
Step: 1880, Train_loss:2.81137
Step: 1890, Train_loss:3.27437
Step: 1900, Train_loss:3.51977
Step: 1910, Train_loss:3.22936
Step: 1920, Train_loss:3.44599
Step: 1930, Train_loss:2.9832
Step: 1940, Train_loss:3.62466
Step: 1950, Train_loss:2.53241
Step: 1960, Train_loss:2.50631
Step: 1970, Train_loss:3.22647
Step: 1980, Train_loss:3.72363
Step: 1990, Train_loss:2.817
Step: 2000, Train_loss:2.80538
Step: 2010, Train_loss:2.8563
Step: 2020, Train_loss:3.45918
Step: 2030, Train_loss:2.97635
Step: 2040, Train_loss:2.51963
Step: 2050, Train_loss:4.87512

**Comment by shekkizh** [comment]
Hi, not sure how much 10mins translates in terms of images trained on. Remember that we are training with very small batch size. For reference look at the training loss plot in Readme. I don't think this is an issue.
